spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/data/web/SpringDataWebProperties.java

```diff
@@ -59,6 +59,11 @@ public Sort getSort() {
 		 */
 		private int defaultPageSize = 20;
 
+		/**
+		 * Use one indexed parameters.
```

Comment:
Can you please expand the javadoc so that it actually explains what that parameter does? The javadoc of the setter is quite complete so a summary of that would be ideal.